tests/unit/integrations/schema-blocks-test.php

```diff
@@ -40,7 +40,7 @@ public function setUp() {
 	public function test_get_conditionals() {
 		static::assertSame(
 			[
-				Schema_Blocks_Conditional::class
+				Schema_Blocks_Conditional::class,
```

Comment:
```suggestion
				Schema_Blocks_Conditional::class
```

Reply:
I think this breaks some browsers

Reply:
Hmm, but this was fixed in the automatic code style fixer, so if we leave it now someone else will fix it the next time. 

Can you tell me more about how this might break some browsers? Should we be looking to make changes to the automatic CS fixer?

Reply:
Also, does this remark about CS mean that the rest of the PR is approved?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas apparently it's ok these days. I'm old :-p

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas: 
"Starting with ECMAScript 5, trailing commas in object literals are legal as well"